citi-onboarding/celeiro-dos-bambas
# Feature(feature/rename-client-folder)

## What I did

- I've  renamed the client folder for src folder 

## How to test

Copy the link in the github from "celeiro-de-bambas-server" 
Go to the terminal 
Create a folder for the server project
Go to the folder
Git clone <link> 

Then 

In the folder "celeiro-de-bambas-server"

1.1. yarn install
1.2. yarn develop
1.3. Check if you are loged

In the folder "celeiro-dos-bambas"

2.1 yarn install
2.2 yarn start

Check if the client folder was rename for src folder 


Test the things that you think are worth testing, even the ones that are not in this description
